### PR TITLE
DOC: fix some doc build warnings/errors

### PR DESCRIPTION
**Pull request by jorisvandenbossche**
Some follow-ups on recent doc PRs

### CODE REVIEW

File: doc/source/missing_data.rst [hunk 1]
```diff
@@ -759,12 +759,7 @@ the ``dtype="Int64"``.
 
 .. ipython:: python
 
-   s = pd.Series(np.random.randn(5), index=[0, 2, 4, 6, 7],
-                 dtype="Int64")
-   s > 0
-   (s > 0).dtype
-   crit = (s > 0).reindex(list(range(8)))
-   crit
-   crit.dtype
```
Comment (jorisvandenbossche) [inline doc/source/missing_data.rst]: @TomAugspurger was there something specific you wanted to show with the above code? It seems you copied this from above about booleans becoming object, but which doesn't seem relevant for integer NA

(the actual fix here is the first line, you can't convert floats from random.randn to Integers)

Comment (TomAugspurger) [reply doc/source/missing_data.rst]: I don't recall anything specific goal behind that example.

**Review by TomAugspurger — APPROVED** [review]

**Comment by jreback** [comment]
lgtm.

**Comment by jreback** [comment]
https://dev.azure.com/pandas-dev/pandas/_build/results?buildId=6334 lint issues

**Comment by jreback** [comment]
thanks @jorisvandenbossche

**Pull request merged**